coq-lsp.opam

```diff
@@ -10,34 +10,38 @@ homepage: "https://github.com/ejgallego/coq-lsp"
 dev-repo: "git+https://github.com/ejgallego/coq-lsp.git"
 authors: [
   "Emilio Jesús Gallego Arias <e@x80.org>"
+  "Ali Caglayan <alizter@gmail.com>"
+  "Shachar Itzhaky <shachari@cs.technion.ac.il>"
+  "Ramkumar Ramachandra <r@artagnon.com>"
 ]
 license: "LGPL-2.1"
 doc: "https://ejgallego.github.io/coq-lsp/"
 
 depends: [
   "ocaml"        {         >= "4.12.0" }
-  "dune"         { build & >= "3.5.0"  }
+  "dune"         { build & >= "3.6.1"  }
```

Comment:
Why this change, in fact our minimal supported dune is 3.2, we use 3.5 only because of the vendoring.